GTFS.DB.PostgreSQL/Collections/PostgreSQLCalendarCollection.cs

```diff
@@ -178,6 +178,35 @@ public bool Remove(string entityId)
             }
         }
 
+        /// <summary>
+        /// Removes a range of entities by their IDs
+        /// </summary>
+        /// <param name="entityIds"></param>
+        /// <returns></returns>
+        public void RemoveRange(IEnumerable<string> entityIds)
+        {
+            using (var command = _connection.CreateCommand())
```

Comment:
Does this work? I have a feeling it won't. I had the same issue with the remove range. I refactored it so that the transactin was outermost, then the foreach, then the command. Please test and let me know if this working or not

Reply:
Cool, I'll try do some tests at some point